src/common/_modules/platform_service_bus/main.tf

```diff
@@ -0,0 +1,22 @@
+module "platform_service_bus_namespace" {
+  source  = "pagopa-dx/azure-service-bus-namespace/azurerm"
+  version = "~> 0.0"
+
+  environment = {
+    prefix          = var.prefix
+    env_short       = "p"
+    location        = var.location
+    app_name        = "platform"
+    instance_number = "01"
+  }
+
+  resource_group_name = var.resource_group_internal
+
+  subnet_pep_id                        = azurerm_subnet.platform_service_bus_namespace_subnet.id
+  private_dns_zone_resource_group_name = var.resource_group_internal
```

Comment:
This should be a reference to the underlying value of the variable `var.resource_groups.event` used in the file `private_dns_zones.tf`

Reply:
thank you, done in https://github.com/pagopa/io-infra/commit/e145a80ebc29b877f695c1e3fcc002c503c8f1fa. that file you mentioned is in another module but I made a variable for it.